exercises/node-javascript/add-esdk-start/store.js

```diff
@@ -0,0 +1,76 @@
+/* ADD-ESDK-START:
```

Comment:
@mattsb42-aws @lizroth Here is the start store file.  The ESDK needs to be added.